server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -930,6 +959,23 @@ public boolean performConsumerUpdates(Consumer updated, Consumer toUpdate,
             }
         }
 
+        if (updated.getContentAccessMode() != null &&
+            !updated.getContentAccessMode().equals(toUpdate.getContentAccessMode()) &&
+            toUpdate.getType().isManifest()) {
+            if (!StringUtils.isBlank(updated.getContentAccessMode()) &&
```

Comment:
Saw this `!StringUtils.isBlank...` predicate earlier.  Let's extract that bit out into a common method.  Makes the code easier to read and cuts down on redundancy.